examples/interoperability/app.js

```diff
@@ -0,0 +1,125 @@
+import 'babel-polyfill';
+/* global document, window,*/
+/* eslint-disable no-console */
+import React, {PureComponent} from 'react';
+import {render} from 'react-dom';
+import DeckGL, {
+  ScatterplotLayer, OrthographicViewport, COORDINATE_SYSTEM
+} from '../../src';
```

Comment:
using npm run start-local helped me resolved multiple copies of luma.gl, but react still complains. maybe I misconfigured something, will give it another try.